server/channels/api4/team.go

```diff
@@ -1672,6 +1673,62 @@ func inviteGuestsToChannels(c *Context, w http.ResponseWriter, r *http.Request)
 	auditRec.Success()
 }
 
+func requestGuestEasyLogin(c *Context, w http.ResponseWriter, r *http.Request) {
+	if c.App.Channels().License() == nil {
```

Comment:
Should we look for a specific license tier? I remember something like `model.MinimumEnterpriseLicense(c.App.Channels().License())`